template.distillery/demo_calendar.eliom

```diff
@@ -21,6 +21,16 @@ let%server service =
 let%client service = ~%service
 
 (* A reactive value containing the currently selected date *)
+(* NOTE: in this example, we want the signal to be stored on the server side,
+ * so that any change will be sent to the server. Only declaring this signal as
+ * shared won't work since you'll end up with two different signals (one for
+ * each side): if you only use a Reactive `map` in `page`, it will use the
+ * server's signal when it's first generated on the server, while the
+ * client-side click event will use its own `f`, so nothing will actually
+ * happen. You can observe this duplication by replacing the `%server` below
+ * with `%shared` and the error the compiler produces: the signal's type can't
+ * be inferred since once instance is never used throughout this program.
+ *)
```

Comment:
The rest looks good. Could you just remove the stars in the beginning of each line? We generally avoid them because they make re-indenting the code harder.